test/libyul/evmCodeTransform/stackReuse/last_use_in_nested_block.yul

```diff
@@ -2,9 +2,11 @@
 // ====
 // stackOptimization: true
 // ----
-// PUSH1 0x0
-// DUP1
-// POP
-// POP
-// PUSH1 0x1
-// POP
+//     /* "":11:12   */
+//   0x00
+//     /* "":15:21   */
+//   pop
+//     /* "":33:34   */
+//   0x01
+//     /* "":0:36   */
+//   stop
```

Comment:
Hmm, is this a new feature that it won't try to clean up the stack? :)

Reply:
Rather a fluke of these weird test cases. Usually it will clean always clean up the stack (actually that's the first thing I'd intend to improve in this stuff - the new code transform has silly unneeded ``pop``s before reverting, etc. - might actually be a good part of the slight increase in code size).

Here it's just that the last operation in the block wants the ``1`` on stack, but doesn't consume it.  The exit layout of the outermost block is empty, but there's no need to actually produce it, i.e. actually pop.